### PR TITLE
VZ-7030 and VZ-7031: Remove velero cli command and modify kubectl exec cmd

### PR DESCRIPTION
**Pull request by theabmitra**

### CODE REVIEW

**Review by davisgomes — APPROVED** [review]

File: content/en/docs/setup/backup/backup.md [hunk 1]
```diff
@@ -115,10 +115,13 @@ After the backup is executed, you can see the hook logs using the `velero backup
 
 ```shell
 # To display the logs from the backup, execute the following command
-$ velero backup logs verrazzano-opensearch-backup -n verrazzano-backup
+$ kubectl logs -n verrazzano-backup -l app.kubernetes.io/name=velero
 
-# To examine the hook logs, exec into the pod as shown
-$ kubectl exec -it vmi-system-es-master-0 -n verrazzano-system -- cat /tmp/verrazzano-backup-hook-1681009483.log
+# Fetch the logfile name as shown below
```
Comment (rosemarymarano) [inline content/en/docs/setup/backup/backup.md:120]: logfile -> log file 
as shown below -> as shown

File: content/en/docs/setup/backup/backup.md [hunk 2]
```diff
@@ -115,10 +115,13 @@ After the backup is executed, you can see the hook logs using the `velero backup
...
+$ kubectl exec -it vmi-system-es-master-0 -n verrazzano-system -- ls -al /tmp | grep verrazzano-backup-hook | tail -n 1 | awk '{print $NF}'
+
+# To examine the hook logs, exec into the pod as shown, and use the filename retrieved above
```
Comment (rosemarymarano) [inline content/en/docs/setup/backup/backup.md:123]: the filename retrieved above -> the file name retrieved previously

File: content/en/docs/setup/backup/restore.md [hunk 3]
```diff
@@ -134,12 +134,15 @@ After the restore operation is executed, you can see the hook logs using the `ve
   <summary>OpenSearch restore logs</summary></summary>
 
 ```shell
-
 # To display the logs from the restore, execute the following command
-$ velero restore logs verrazzano-opensearch-restore -n verrazzano-backup
+$ kubectl logs -n verrazzano-backup -l app.kubernetes.io/name=velero
+
+# Fetch the logfile name as shown below
```
Comment (rosemarymarano) [inline content/en/docs/setup/backup/restore.md:140]: logfile name as shown below -> log file name as shown

File: content/en/docs/setup/backup/restore.md [hunk 4]
```diff
@@ -134,12 +134,15 @@ After the restore operation is executed, you can see the hook logs using the `ve
...
+$ kubectl exec -it vmi-system-es-master-0 -n verrazzano-system -- ls -al /tmp | grep verrazzano-restore-hook | tail -n 1 | awk '{print $NF}'
+
+# To examine the hook logs, exec into the pod as shown, and use the filename retrieved above
```
Comment (rosemarymarano) [inline content/en/docs/setup/backup/restore.md:143]: the filename retrieved above -> the file name retrieved previously

**Review by rosemarymarano — COMMENTED** [review]
Few minor edits

File: content/en/docs/setup/backup/restore.md (hunk 4 above)
Comment (theabmitra) [reply content/en/docs/setup/backup/restore.md]: done

File: content/en/docs/setup/backup/restore.md (hunk 3 above)
Comment (theabmitra) [reply content/en/docs/setup/backup/restore.md]: done

File: content/en/docs/setup/backup/backup.md (hunk 2 above)
Comment (theabmitra) [reply content/en/docs/setup/backup/backup.md]: done

File: content/en/docs/setup/backup/backup.md (hunk 1 above)
Comment (theabmitra) [reply content/en/docs/setup/backup/backup.md]: done

**Review by MelwynPaul — APPROVED** [review]
Verified that all the doc comments from Rosemary are fixed. 
Approved.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**